### ISSUE TITLE
add persistence to save payment methods after app restart

### ISSUE THREAD

**Issue by odoric27** [devops-s17-payments/payments#12]
*As a* User
*I need* a way to save payment methods
*So that* when I restart the app I can still access my payments

**Assumptions**
* ~~using redisDB (see previous implementations provided by Professor for template)~~
* using postgres, sqlalchemy

**Acceptance Criteria**

*Given* a user with three (3) payments
*When* I issue GET `/payments`
*Then* I should see an array of 3 payment methods
*And* when I restart the app
*Then* I should still have 3 payment methods

**Comment by odoric27** [comment]
i'm hitting a wall with db setup. the branch 'add_sqlalchemy' is where i left things for now if anyone wants to take a look and help me figure this out.

this branch includes the updated vagrantfile, requirements for db/orm support as well as the latest revised models class. there's a file called 'initdb.py' that should create the tables from the models and add in some sample data, but can't get it working. doesn't throw any errors when it runs, but it doesn't do anything.

i had to do some manual initialization of the postgres db prior to running initdb above so just fetching this branch won't give you everything you need. likely that i screwed something up in this process so I'm thinking my next attempt will be to do something similar to this repo (https://github.com/jackdb/pg-app-dev-vm/blob/master/Vagrantfile) to init the db via vagrant, but its unclear whether or not this is what's even causing other issues.

so in short, i need to take a break from this for a minute but if anyone has thoughts/ideas, please share.

**Comment by odoric27** [comment]
looking through the backlog and this got tied up in  #96. i'll close this out when existing PRs go through

**Comment by odoric27** [comment]
as noted above, this is now closed per #96
